datadog/tests/data_source_datadog_logs_archives_order_test.go

```diff
@@ -0,0 +1,24 @@
+package test
+
+import (
+	"context"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDatadogLogsArchivesOrderDatasource(t *testing.T) {
+	_, accProviders := testAccProviders(context.Background(), t)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: accProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: `data "datadog_logs_archives_order" "order" {}`,
```

Comment:
Nit: It would be nice to create an archive and assert that its id is in the list.